packages/core/src/electron-browser/window/electron-window-service.ts

```diff
@@ -85,4 +100,15 @@ export class ElectronWindowService extends DefaultWindowService {
         });
         return response === 0; // 'Yes', close the window.
     }
+
+    /**
+     * Updates the window zoom level based on the value set in `Preferences`.
+     */
+    protected updateWindowZoomLevel(): void {
+        const preferredZoomLevel = this.windowPreferences['window.zoomLevel'];
+        const webContents = remote.getCurrentWindow().webContents;
+        if (webContents.getZoomLevel() !== preferredZoomLevel) {
+            webContents.setZoomLevel(preferredZoomLevel);
+        }
```

Comment:
Just a question, at this point we update when a new value is set, so is `webContents.getZoomLevel() !== preferredZoomLevel` always truthy? I also don't think its an issue of if we `setZoomLevel` always but this might be cleaner.